bundles/io.openliberty.tools.eclipse.ui/src/io/openliberty/tools/eclipse/DevModeOperations.java

```diff
@@ -1060,6 +1060,22 @@ public static Path getGradleTestReportPath(String projectPath) {
         return path;
     }
 
+	/**
+	 * Returns the path of the xml file containing app monitoring configuration.
+	 *
+	 * @param projectPath The project's path.
+	 *
+	 * @return The custom path of the xml file containing the config to
+	 *         enable/disable app monitoring.
+	 */
+	public static Path getMavenXmlFilePathInOverridesDirectory(String projectPath) {
```

Comment:
We should probably have this method in the LibertyPluginSWTBotDebuggerTest class since it is only used by the tests.